modern/src/SocketController.js

```diff
@@ -123,6 +164,11 @@ const SocketController = () => {
           message={notification.message}
           autoHideDuration={snackBarDurationLongMs}
           onClose={() => setEvents(events.filter((e) => e.id !== notification.id))}
+          ContentProps={{
+            sx: {
+              background: 'red',
+            },
+          }}
```

Comment:
Let's move out all unrelated changes like this.

Reply:
My bad